request/auth.php

```diff
@@ -14,7 +14,7 @@
                     case 'login':
                         if (!empty($_POST['email']) && !empty($_POST['password'])) {
                             if (FirstAide\Utils::isValidEmail($_POST['email'])) {
-                                $Auth = FirstAide\Authentication::withEmailPassword(
+                                $Auth = FirstAide\Authentication::createInstanceWithEmailPassword(
```

Comment:
@fatimarafiqui What I suggested was to move all the contents of the *if condition* outside to a method so that you will just have to call if(isValid($_POST)) and that method will do all the checks and return a boolean. Makes the code cleaner. 

```
boolean isValidCSRFToken($_POST) {
     return ( isset($_POST) && isset($_POST['csrf_token']) && $_POST['csrf_token'] == $csrf_token )
}
```
You can call this method inside the if condition.